jenkins-integration/jenkins-jobs/common/scripts/job-steps/035_customize_hocon_settings.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+pushd jenkins-integration
+source jenkins-jobs/common/scripts/job-steps/initialize_ruby_env.sh
+
+set -x
+set -e
+
+# Setup SSH agent for SSH access to SUT
+eval $(ssh-agent -t 24h -s)
+ssh-add ${HOME}/.ssh/id_rsa
+
+bundle exec beaker \
+        --config hosts.yaml \
+        --load-path lib \
+        --log-level debug \
+        --no-color \
+        --tests \
+beaker/install/shared/35_customize_hocon_settings.rb
+#beaker/install/shared/99_restart_server.rb
```

Comment:
I wonder if we should create some sort of rollup restart / reload server step before we do the run_gatling_sim step?  Seems pretty likely that we'll want to have the server restarted after reconfiguring some hocon settings.  Without a way to configure this, though, not sure there's a guarantee that the server will have been restarted and, therefore, using the new settings before the gatling run is kicked off.